source/_drafts/The-Importance-of-Metadata.md

```diff
@@ -0,0 +1,18 @@
+---
+title: The Importance of Metadata
+author: Mukund Srinivas
+date:
+categories: general
+number:
+tags:
+---
+Agricultural technology(Agri-Tech) is a portfolio of products and serivices that help the agricultural industry, these could range from products that help in reducing operational costs of the client(developing algorithms for efficiently moving and storing food), to make policy decisions (What are the crops that need a subsidy that particular season).
+Working at an  Agri-Tech consultancy has taught me a few things about data and drawing inferences from it. The most profound of them is the ability to tell the most useful piece of information from the least . It might happen that the most significant piece of information is actually the metadata. It is a file that is auxillary to the main file and typically contains information describing the data  that might be helpful in answering rudimentary questions about the contents of the main file.
+
+For example, A census dataset of a province might have a metadata that describes the number of respondents , the type of the data itself (csv,txt) , the source of the dataset (survey year, survey type), the methology of collection(internet survey, ground based survey). A good analogy is if data is the textbook, the metadata is its table of contents.
```

Comment:
I dont think this paragraph is relevant, sure it explains metadata more but brining in census data seems to confuse what you are trying to say